jbmc/src/java_bytecode/java_bytecode_convert_method.cpp

```diff
@@ -2181,7 +2181,8 @@ exprt::operandst &java_bytecode_convert_methodt::convert_const(
       is_float ? ieee_float_spect::single_precision()
                : ieee_float_spect::double_precision());
 
-    ieee_floatt value(spec);
+    auto rm = ieee_floatt::rounding_modet::ROUND_TO_EVEN;
+    ieee_floatt value(spec, rm);
```

Comment:
This one is surprising: why is this not an `ieee_float_valuet`?

Reply:
The call to `from_integer` may need to round. I'll add a comment.